src/azure/Model/CodeModelPya.cs

```diff
@@ -30,9 +30,9 @@ public class CodeModelPya : CodeModelPy
                     m.Extensions.ContainsKey(AzureExtensions.LongRunningExtension) &&
                     (bool) m.Extensions[AzureExtensions.LongRunningExtension]);
 
-        public bool HasAnyCloudErrors =>
+        public bool HasAnyHttpResponseErrors =>
             MethodTemplateModels.Any(item =>
-                (item.DefaultResponse.Body == null) || (item.DefaultResponse.Body.Name == "CloudError"));
+                (item.DefaultResponse.Body == null) || (item.DefaultResponse.Body.Name == "HttpResponseError"));
```

Comment:
I'm in favor of killing this weird thing item.DefaultResponse.Body.Name == "HttpResponseError"
That's not a feature, that's old legacy